lib/msal-node/README.md

```diff
@@ -88,7 +88,7 @@ Any major MSAL Node release:
 
 | MSAL Node version | MSAL support status | Supported Node versions |
 | ----------------- | ------------------- | ----------------------- |
-| 2.x.x             | Active development  | 18, 20                  |
+| 2.x.x             | Active development  | 16, 18, 20              |
 | 1.x.x             | In maintenance      | 10, 12, 14, 16, 18      |
```

Comment:
Is it clarified anywhere what "In maintenance" means? Can be rephrase to "Deprecated"? Or add a note below, something like: "MSAL Node 2.x is in active development and receives all bug fixes and any new features. MSAL Node 1.x is deprecated and will only receive high security bug fixes (and no new features). Our recommendation is to upgrade to the latest MSAL Node 2.x in your applications."

Reply:
I will leave it to @EmLauber to wordsmith this communication